src/main/java/com/stripe/param/PlanCreateParams.java

```diff
@@ -532,11 +532,13 @@ public Builder setUsageType(UsageType usageType) {
     String name;
 
     /**
-     * An arbitrary string to be displayed on your customer's credit card statement. This may be up
-     * to 22 characters. The statement description may not include "' characters, and will appear on
-     * your customer's statement in capital letters. Non-ASCII characters are automatically
-     * stripped. While most banks display this information consistently, some may display it
-     * incorrectly or not at all.
+     * An arbitrary string to be displayed on your customer's credit card or bank statement. While
+     * most banks display this information consistently, some may display it incorrectly or not at
+     * all.
+     *
+     * <p>This may be up to 22 characters. The statement description may not include `<`, `>`, `\`,
```

Comment:
The new docstring causes the build to fail because the `<` and >` characters need to be HTML encoded to `&lt'` and `&gt;`. We'll need to fix this in codegen. cc @richardm-stripe